<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugPreference</class>
 <widget class="QWidget" name="DebugPreference">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Debug Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="scannerGroupBox">
     <property name="title">
      <string>Virtual Scanner Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="mockFileLabel">
        <property name="text">
         <string>Mock File Path:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="filePathLayout">
        <item>
         <widget class="QLineEdit" name="mockFilePathEdit">
          <property name="placeholderText">
           <string>Select mock data file path</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browseButton">
          <property name="text">
           <string>Browse...</string>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="scanTimeLabel">
        <property name="text">
         <string>Scan Time (seconds):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="scanTimeSpinBox">
        <property name="suffix">
         <string> sec</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="enableDelayLabel">
        <property name="text">
         <string>Enable Scan Delay:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="enableDelayCheckBox">
        <property name="text">
         <string>Simulate real scan time</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="loggingGroupBox">
     <property name="title">
      <string>Logging Settings</string>
     </property>
     <layout class="QFormLayout" name="loggingFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="logLevelLabel">
        <property name="text">
         <string>Log Level:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="logLevelComboBox">
        <item>
         <property name="text">
          <string>Debug</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Info</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Warning</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Error</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Critical</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="logToFileLabel">
        <property name="text">
         <string>Log to File:</string>
        </property>
       </widget>
      </item>
             <item row="1" column="1">
        <widget class="QCheckBox" name="logToFileCheckBox">
         <property name="text">
          <string>Enable file logging</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="logFilePathLabel">
         <property name="text">
          <string>Log File Path:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="logFilePathLayout">
         <item>
          <widget class="QLineEdit" name="logFilePathEdit">
           <property name="placeholderText">
            <string>Select log file storage path</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="browseLogPathButton">
           <property name="text">
            <string>Browse...</string>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
